test/e2e/scenarios/Cluster/cluster_terminal_condition.yaml

```diff
@@ -32,6 +32,9 @@ steps:
         name: cluster$RANDOM_SUFFIX
       spec:
         nodeType: db.t4g.small
+        aclName: open-access
+        numShards: 1
+        numReplicasPerShard: 0
```

Comment:
This should not be added whole idea is test should run once you fix the spec that has issue?

Reply:
Yeah the root reason is that test account is full now. Deleting some clusters in account.